base/common/src/org/dogtagpki/tps/apdu/SelectAPDU.java

```diff
@@ -30,6 +30,18 @@ public SelectAPDU(byte p1, byte p2, TPSBuffer theData)
         setP1(p1);
         setP2(p2);
         setData(theData);
+        // Add trailer byte
+        TPSBuffer trailer = new TPSBuffer(1);
+        setTrailer(trailer);
```

Comment:
how about these? he old cards work fine with this change?

Reply:
Yes.